docs/.vitepress/config.ts

```diff
@@ -1,21 +1,21 @@
 import { defineConfig } from 'vitepress'
-import markdownItTaskCheckbox from 'markdown-it-task-checkbox'
 
-export default {
+export default defineConfig({
   base: '/website/',
-  title: 'ICC CE | 智能课堂绘图增强工具',
+  title: 'ICC CE',
```

Comment:
在 `docs/.vitepress/config.ts` 中，您更新了 `title` 字段。建议确保这个标题在所有页面上都保持一致性和准确性。

```suggestion
title: 'ICC CE | 智能课堂绘图增强工具',